tasks/orca_recovery_adapter/test/integration_tests/integration_test_orca_recovery_adapter.py

```diff
@@ -1,93 +1,125 @@
+                        }
+                    ]
+                }
+            },
+            "cumulus_meta": {
+                "system_bucket": system_bucket_name
+            }
         }
 
         expected_output = {
-          "payload": {
-            "granules": [
-              {
-                "granuleId": granule_id,
-                "createdAt": createdAt_time,
-                "files": [
-                  {
-                    "bucket": cumulus_bucket_name,
-                    "key": key_name
-                  }
-                ]
-              }
-            ],
-            "copied_to_orca": [
-              "s3://" + cumulus_bucket_name + "/" + key_name,
-            ]
-          },
-          "task_config": {
-            "providerId": provider_id,
-            "executionId": execution_id,
-            "collectionShortname": collection_shortname,
-            "collectionVersion": collection_version,
-            "defaultBucketOverride": orca_recovery_bucket
-          },
-          "exception": "None"
-        }
-        context = Mock()
-        result = orca_recovery_adapter.handler(orca_recovery_adapter_input_event, context)
-        self.assertEqual(result, expected_output)
-        # verify that the object exists in recovery bucket
-        try:
-            head_object_output = boto3.client("s3").head_object(
-                Bucket=orca_recovery_bucket, Key=key_name)
-            self.assertEqual(
-                200,
-                head_object_output["ResponseMetadata"]["HTTPStatusCode"],
-                f"Error searching for object {key_name} in the {orca_recovery_bucket}",
-            )
-        except Exception as ex:
-            raise ex
+            'payload': {
+                'granules': [{
+                    'granuleId': granule_id,
+                    'keys': [{
+                        'key': key_name, 'destBucket': cumulus_bucket_name
+                    }],
+                    'recoverFiles': [{
+                        'success': True,
+                        'filename': file_name,
+                        'keyPath': key_name,
+                        'restoreDestination': cumulus_bucket_name,
+                        's3MultipartChunksizeMb': None,
+                        'statusId': 1,
+                        'requestTime': mock.ANY,
+                        'lastUpdate': mock.ANY
+                    }]
+                }],
+                'asyncOperationId': mock.ANY
+            },
+            'task_config': orca_recovery_adapter_input_event["task_config"],
+            'meta': orca_recovery_adapter_input_event["meta"],
+            'cumulus_meta': orca_recovery_adapter_input_event["cumulus_meta"],
+            'exception': 'None'}
+
+        # noinspection PyTypeChecker
+        result = orca_recovery_adapter.handler(
+            orca_recovery_adapter_input_event,
+            lambda_context(),
+        )
+        self.assertEqual(expected_output, result)
```

Comment:
Should we also include some verification to validate if files are present in the bucket after recovery?

Reply:
Not viable, unfortunately. Since all this does is _request_ that the files be recovered, they will not be anywhere different after the 'Recovery' flow is complete. We'll need to do a rename at some point.

That said, since this just checks the integration layer, we're fine. The test still makes sure that this code successfully communicates with our ORCA code.